topics/reference_guide/custom_language_support/websymbols.md

```diff
@@ -0,0 +1,822 @@
+         "unique": true,
+         "required": false
+       }
+     ]
+   },
+   "ng-event-plugins-modifiers": [
+     {
+       "name": "prevent"
+     },
+     {
+       "name": "stop"
+     },
+     ...
+   ]
+ }
+]
+```
+{lang="JSON" collapsible="true" default-state="collapsed"}
+
+#### Vue
+
+Vue plugin Web Types are available here for reference.
+Any Web Types file targeting only Vue support should have `framework` property set to `vue`.
+Highlights:
+
+{style="full"}
+/html/vue-components
+: Use `/html/vue-components` to contribute Vue components.
+
+/html/vue-directives
+: Use `/html/vue-directives` to contribute Vue directives.
+Use `attribute-value` property to specify the type of value expression. E.g.:
+```
+"attribute-value": {
+  "type": "boolean",
+  "required": true
+}
+```
+{lang="JSON" collapsible="true" default-state="collapsed"}
+
+/html/vue-file-top-elements
+: Use `/html/vue-file-top-elements` to contribute any custom top-level elements available in Vue Single Component File
+
+A Vue `/html/vue-components` contribution supports:
+
+{style="full"}
+/html/props
+: Use `/html/props` to contribute Vue component props, e.g:
+```
+"props": [
+ {
+  "name": "appear",
+  "description": "Whether to apply transition on initial render.",
+  "type": "boolean",
+  "default": "false"
+ }
+]
+```
+{lang="JSON" collapsible="true" default-state="collapsed"}
+
+/html/slots
+: Use `/html/slots` to contribute Vue component slots.
+For scoped slots use `vue-properties` to provide list of scoped slot properties.
+Example:
+```
+"slots": [
+ {
+   "name": "img",
+   "description": "Expects the [v-img](/components/images) component.",
+   "doc-url": "https://www.vuetifyjs.com/api/v-app-bar#slots",
+   "vue-properties": [
+     {
+       "name": "props",
+       "type": "{ height: string, src: string | srcObject }"
+     }
+   ]
+ }
+]
+```
+{lang="JSON" collapsible="true" default-state="collapsed"}
+
+/js/events
+: Use `/js/events` to contribute Vue component events, e.g.:
+```
+"js": {
+ "events": [
+   {
+     "name": "input",
+     "description": "The updated bound model"
+   }
+ ]
+}
+```
+{lang="JSON" collapsible="true" default-state="collapsed"}
+
+html/vue-model
+: Use `html/vue-model` to contribute settings for Vue model directive. E.g.:
+```
+"vue-model": {
+  "prop": "show",
+  "event": "input"
+}
+```
+{lang="JSON" collapsible="true" default-state="collapsed"}
+
+
+A Vue `/html/vue-directives` contribution supports:
+
+{style="full"}
+/html/argument
+: Use `/html/argument` as a Vue directive argument. E.g.:
+```
+"argument": {
+ "name": "attribute or property name",
+ "description": "Optional attribute or property name",
+ "pattern": {
+   "items": [
+     {
+       "path": "/html/attributes",
+       "includeVirtual": false
+     }
+   ]
+ }
+}
+```
+{lang="JSON" collapsible="true" default-state="collapsed"}
+
+
+/html/modifiers
+: Use `/html/modifiers` as a Vue directive modifier. E.g.:
+```
+"modifiers": [
+ {
```

Comment:
Please add a 2-spaces indent. Currently, it renders no indent for `{` and the rest.